raster_tester/aligned.py

```diff
@@ -10,7 +10,7 @@ def tiled(sources, equal_blocks=True):
         for source in sources:
             with rasterio.open(source) as src:
 
-                if not src.profile['tiled']:
+                if not src.is_tiled:
```

Comment:
👌 `is_tiled`. After running into a tiling/profile related bug in rio-alpha, I really appreciate this dataset property. Got to use this more often.